src/App.js

```diff
@@ -9,7 +12,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <main className="main">
+    <Article
+      title={article.title}
+      date={article.date}
+      text={article.text}
+    />
```

Comment:
Easier way to pass all object properties.

```suggestion
    <Article {...article} />
```